src/orange/OrangeData.hh

```diff
@@ -273,6 +279,9 @@ struct OrangeParamsData
     Items<SurfaceType> surface_types;
     Items<Connectivity> connectivities;
     Items<VolumeRecord> volume_records;
+    Items<BoundingBox> bboxes;
```

Comment:
In a `k-D` tree the bounding boxes of the leaves are implicitly defined by their location in the hierarchy: the BIH should be the same. I don't think we need to have the full bboxes for every volume on GPU.

Reply:
This is an interesting research question, maybe one we should discuss offline. I agree that keeping the bounding boxes is unnecessary for k-D trees, but I am not sure it is unnecessary for BIH tress. With a k-D tree, going down an edge means that a given point is contained within a finite cuboid. This cuboid gets smaller on each level down you go. With BIH, going down an edge means a given point in within some infinite half-space. If you reach a leaf that is N levels down, the intersection of the N half spaces could still be huge, even infinite. In other words, when you reach a leaf in a BIH tree, there might still be only a small chance you are actually in a volume. In this case, it makes sense to test the volume's bbox first.